pkg/datapath/linux/probes/probes.go

```diff
@@ -40,6 +43,14 @@ var (
 	probeManager *ProbeManager
 )
 
+// ErrKernelConfigNotFound represents the error if kernel config is unavailable
+// to the cilium agent.
+type ErrKernelConfigNotFound struct{}
+
+func (e *ErrKernelConfigNotFound) Error() string {
+	return "CONFIG_BPF kernel parameter not found"
```

Comment:
No longer true. Maybe something like:
```suggestion
	return "Kernel config file not found"
```